src/status/migrations/0002_location_height.py

```diff
@@ -14,7 +14,7 @@ class Migration(migrations.Migration):
             name="height",
             field=models.DecimalField(
                 decimal_places=3,
-                help_text="Height of the sensor in meters above sea level.",
+                help_text="Altitude of the sensor in meters above the WGS84 ellipsoid.",
```

Comment:
This may be a little confusing as altitude above sea level or ground level is typically used. I am guessing altitude above the WGS84 ellipsoid is more like sea level? Do you think this could be clarified?

Reply:
In general, is there a conversion that needs to happen to go from above mean sea level to above WGS84 ellipsoid or is this match what a user would typically get from GPS or Google Earth/Maps?

Reply:
> In general, is there a conversion that needs to happen to go from above mean sea level to above WGS84 ellipsoid or does this match what a user would typically get from GPS or Google Earth/Maps?

See above regarding GPS. After some quick research, it's not entirely clear what Google Earth uses. The best I could find is that their API uses the same GeoJSON standard as the SigMF spec (see the geometry object [here](https://developers.google.com/earth-engine/reference/rest/v1/projects.locations.assets)). In any case, Google Earth/Maps is not as reliable or accurate source of elevation data compared to an actual GPS unit. If a user inputs a value from one of these sources, it is probably going to be close but not accurate anyways.

Reply:
Sounds good, let's just leave it as is for now.